js/views/modals/purchase/Complete.js

```diff
@@ -0,0 +1,126 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseVw from '../../baseVw';
+import ChatMessage from '../../../models/chat/ChatMessage';
+import { getEmojiByName } from '../../../data/emojis';
+
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    this.processingTime = this.options.listing.get('item').processingTime ||
+      app.polyglot.t('purchase.completeSection.noData');
+    this.vendorPeerID = this.options.listing.get('vendorID').peerID;
+    this.orderID = ''; // supplied by pending.js when the order is paid for
+    this.state = 'pendingPayment';
+  }
+
+  className() {
+    return 'complete';
+  }
+
+  events() {
```

Comment:
Can you prevent default on keydown?

```
  onKeyDownMessageInput(e) {
    if (!e.shiftKey && e.which === 13) e.preventDefault();
  }
```

That way, the cursor won't jump to the next line before the message is sent.